src/Rs/Json/Patch/Operations/Add.php

```diff
@@ -86,7 +93,7 @@ public function perform($targetDocument)
         if ($get === null && $lastPointerPart !== Pointer::LAST_ARRAY_ELEMENT_CHAR) {
             if (ctype_digit($lastPointerPart) && $lastPointerPart > count($rootGet)) {
                 if ($rootPointer == $lastPointerPart && is_array($targetDocument)) {
-                    if (intval($lastPointerPart) <= count($targetDocument) + 1) {
```

Comment:
Best-practice to use casting instead of intval if you don't need base-conversion